CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.5.2:
+
+##### Breaking
+
+* None.
+
+##### Enhancements
+
+* Improve autocorrect for OpeningBraceRule.
```

Comment:
Please add 2 trailing spaces on the last line describing the change, as outlined in [CONTRIBUTING.md](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md).